application-admintools-default/src/main/resources/templates/downloadTemplate.vm

```diff
@@ -0,0 +1,139 @@
+          </li>
+        </ul>
+      </li>
+    </ul>
+
+    #warning($escapetool.xml($services.localization.render('adminTools.dashboard.section.download.warning')))
+
+  </div>
+
+#else
+  #set($warningMessage =
+    $escapetool.xml($services.localization.render('adminTools.dashboard.section.backend.configuration.error'))+ '<br>')
+  #set($warningMessage = $warningMessage +
+    $escapetool.xml($services.localization.render('adminTools.dashboard.section.backend.supportedServers.error'))
+    + $stringtool.join($services.admintools.getSupportedServers(), ', '))
+
+  #warning($warningMessage)
+#end
+
+<div class="modal fade" id="downloadFilesModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span></button>
+        <h1 class="modal-title" id="myModalLabel" style="text-align: center;">
+          $escapetool.xml($services.localization.render('adminTools.dashboard.modal.download.title'))</h1>
+      </div>
+      <div class="modal-body" style="text-align: center;">
+        <form id="downloadForm" action="$services.admintools.getContextPath()/rest/admintools/files" method="POST">
+          <label><input type="checkbox" name="files" value="xwikiConfigFileDataResource" checked> xwiki.cfg</label><br>
+          <label><input type="checkbox" name="files" value="xwikiPropertiesFileDataResource" checked>
+            xwiki.properties</label><br>
+          <label><input type="checkbox" name="files" value="dataProviderResource" checked>
+            $escapetool.xml($services.localization.render('adminTools.dashboard.modal.download.provided'))</label><br>
+          <label><input type="checkbox" name="files" value="logsDataResource" checked> Logs</label><br>
+          <div id="dateFields">
+            <label>$escapetool.xml($services.localization.render('adminTools.dashboard.modal.download.date.from'))
+            </label> <input type="date" id="from" name="from">
+            <label>$escapetool.xml($services.localization.render('adminTools.dashboard.modal.download.date.to'))
+            </label> <input type="date" id="to" name="to">
+          </div>
+          <br>
+          <br>
+          <input class="btn btn-primary btn-lg" type="submit"
+            value="$escapetool.xml($services.localization.render('adminTools.dashboard.modal.download.button'))">
+        </form>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-default" data-dismiss="modal">
+          $escapetool.xml($services.localization.render('adminTools.dashboard.modal.button.close'))</button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="modal fade" id="viewLogsModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
```

Comment:
This is the same as the one defined in configurationTemplate?

Reply:
Yes. I'm not sure if it is the best solution, but I moved them in AdminTools WebHome